include/zephyr/dt-bindings/clock/stm32g0_clock.h

```diff
@@ -17,4 +15,79 @@
 #define STM32_PERIPH_BUS_MIN	STM32_CLOCK_BUS_IOP
 #define STM32_PERIPH_BUS_MAX	STM32_CLOCK_BUS_APB1
 
+/** Peripheral clock sources */
+/* RM0444, §5.4.21/22 Clock configuration register (RCC_CCIPRx) */
+
+/** Fixed clocks  */
+#define STM32_SRC_HSI		0x001
+#define STM32_SRC_MSI		0x002
+#define STM32_SRC_LSE		0x003
+#define STM32_SRC_LSI		0x004
+/** System clock */
+#define STM32_SRC_SYSCLK	0x005
+/** Peripheral bus clock */
+#define STM32_SRC_PCLK		0x006
+/** PLL clock */
+#define STM32_SRC_PLLCLK	0x007
+
+#define STM32_SRC_CLOCK_MIN	STM32_SRC_HSI
+#define STM32_SRC_CLOCK_MAX	STM32_SRC_PLLCLK
+
+/**
+ * @brief STM32 clock configuration bit field.
+ *
+ * - reg   (1/2/3)         [ 0 : 7 ]
+ * - shift (0..31)         [ 8 : 12 ]
+ * - mask  (0x1, 0x3, 0x7) [ 13 : 15 ]
+ * - val   (0..7)          [ 16 : 18 ]
+ *
+ * @param reg RCC_CCIPRx register offset
+ * @param shift Position within RCC_CCIPRx.
+ * @param mask Mask for the RCC_CCIPRx field.
+ * @param val Clock value (0, 1, ... 7).
+ */
+
+#define STM32_CLOCK_REG_MASK    0xFFU
+#define STM32_CLOCK_REG_SHIFT   0U
+#define STM32_CLOCK_SHIFT_MASK  0x1FU
+#define STM32_CLOCK_SHIFT_SHIFT 8U
+#define STM32_CLOCK_MASK_MASK   0x7U
+#define STM32_CLOCK_MASK_SHIFT  13U
+#define STM32_CLOCK_VAL_MASK    0x7U
+#define STM32_CLOCK_VAL_SHIFT   16U
+
+#define STM32_CLOCK(val, mask, shift, reg)					\
+	((((reg) & STM32_CLOCK_REG_MASK) << STM32_CLOCK_REG_SHIFT) |		\
+	 (((shift) & STM32_CLOCK_SHIFT_MASK) << STM32_CLOCK_SHIFT_SHIFT) |	\
+	 (((mask) & STM32_CLOCK_MASK_MASK) << STM32_CLOCK_MASK_SHIFT) |		\
+	 (((val) & STM32_CLOCK_VAL_MASK) << STM32_CLOCK_VAL_SHIFT))
+
+/** @brief RCC_CSR register offset */
+#define CCIPR_REG		0x88
```

Comment:
Missing `CCIPR2_REG` or?
Based on `RM0444.pdf` and `RM0454.pdf` the `CCIPR_REG` is `0x54` and `CCIPR2_REG` is `0x58`.